util/src/main/java/eu/darkbot/util/http/Http.java

```diff
@@ -178,12 +179,38 @@ public Http setRawParam(Object key, Object value) {
      */
     @SuppressWarnings("PMD.ArrayIsStoredDirectly")
```

Comment:
probably no longer needed

Reply:
PMD still considers it a violation of the rules